App.tsx

```diff
@@ -1,21 +1,49 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- */
-
 import React from 'react';
-import {Text, View} from 'react-native';
-import {SafeAreaView} from 'react-native-safe-area-context';
+import styled from 'styled-components/native';
+import {Provider} from 'react-redux';
+import {store} from './src/app/store';
+import UserIcon from 'react-native-vector-icons/EvilIcons';
+import {CryptoContainer} from './src/components';
 
 function App(): JSX.Element {
   return (
-    <SafeAreaView>
-      <View>
-        <Text>Hola mundo</Text>
-      </View>
-    </SafeAreaView>
+    <Provider store={store}>
+      <ViewContent>
+        <HeaderContent>
+          <HeaderInfo>
+            <Title>CryptoTracker Pro</Title>
+            <UserIcon name="user" size={60} />
+          </HeaderInfo>
+        </HeaderContent>
+        <CryptoContainer />
+      </ViewContent>
+    </Provider>
   );
 }
 export default App;
+
+const ViewContent = styled.View`
+  background: #fffafa;
```

Comment:
Use theme, styled components allows you to use a themeProvider


https://styled-components.com/docs/advanced

Reply:
Styles for heach component should be stored in another file: 

App.tsx
styles.ts